### PR TITLE
Python2 > 3: GSP185 - App Dev: Storing Image and Video Files in Cloud Storage - Python

### PR DESCRIPTION
**Pull request by siddharthchaurasia-tudip**
- Migrated files to work in python3.

### CODE REVIEW

File: courses/developingapps/python/cloudstorage/start/quiz/api/routes.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import api
+from . import api
```
Comment (mylenebiddle) [inline courses/developingapps/python/cloudstorage/start/quiz/api/routes.py]: let's make this an absolute import, please

File: courses/developingapps/python/cloudstorage/start/quiz/webapp/routes.py [hunk 2]
```diff
@@ -19,14 +19,14 @@
 from flask import send_from_directory
 from flask import request, redirect
 
-import questions
+from . import questions
```
Comment (mylenebiddle) [inline courses/developingapps/python/cloudstorage/start/quiz/webapp/routes.py]: same here

File: courses/developingapps/python/cloudstorage/start/quiz/api/routes.py (hunk 1 above)
Comment (siddharthchaurasia-tudip) [reply courses/developingapps/python/cloudstorage/start/quiz/api/routes.py]: @mylenebiddle , I have commited the requested changes in above python file (routes.py) as required. Please review the changes.

File: courses/developingapps/python/cloudstorage/start/quiz/webapp/routes.py (hunk 2 above)
Comment (siddharthchaurasia-tudip) [reply courses/developingapps/python/cloudstorage/start/quiz/webapp/routes.py]: @mylenebiddle , I have commited the requested changes in above python file (routes.py) as required. Please review the changes.

**Comment by siddharthchaurasia-tudip** [comment]
@mylenebiddle , Please review the changes.

**Pull request merged**